src/clj/puppetlabs/services/jruby/jruby_puppet_internal.clj

```diff
@@ -27,18 +28,13 @@
   classpath.
 
   See also:  http://jruby.org/apidocs/org/jruby/runtime/load/LoadService.html"
-  "puppet-server-lib")
+  "classpath:/puppet-server-lib")
```

Comment:
With this just including the relative path, JRuby will fail to find any of the Puppet Server-specific Ruby files inside of the jar at run-time.  I was able to get this to work, however, by using a prefix of either `classpath:` or `uri:classloader:` to work.  `classpath:` seemed more appropriate since we just want JRuby to find the Puppet Server Ruby files from within the classpath.